apps/web/src/app/(main)/pricing/page.tsx

```diff
@@ -0,0 +1,89 @@
+import type { TabDataProps } from '@/components/ui/animated-tabs'
+import AnimatedTab from '@/components/ui/animated-tabs'
+import { ColorBGSVG } from '@public/hero'
+import type { CardProps } from '@/components/pricing/card'
+import PriceCard from '@/components/pricing/card'
+
+function About(): React.JSX.Element {
+  const tabsData: TabDataProps[] = [
+    { id: 'monthly', label: 'Monthly' },
+    { id: 'yearly', label: 'Yearly', tag: '-20%', special: true }
+  ]
+
+  const priceCardData: CardProps[] = [
```

Comment:
Don't import the Prop interface here, create a type in `types` folder and this use it, and if you think that `CardProps` and `priceCardData` have same property then extend the newly created type in the `CardProps`.

Let's say the type is named `PriceCard` then modify the  `CardProps` as ` interface CardProps extends PriceCard` and `priceCardData` will have the type `const priceCardData: PriceCard[]`

Reply:
I tried to use types now and export just the types. do let me know if thats correct.